ui/imodel-components-react/package.json

```diff
@@ -106,8 +106,7 @@
     "ts-node": "^10.8.2",
     "typemoq": "^2.1.0",
     "typescript": "~5.0.2",
-    "eslint-config-prettier": "~8.8.0",
-    "@rushstack/eslint-patch": "^1.3.0"
+    "eslint-config-prettier": "~8.8.0"
```

Comment:
can you alphabetize deps

Reply:
sure, will do. I used `rush add` to add this deps, i'm surprised it didn't do it automatically. Am I doing it wrong? 🤔

Reply:
i typically edit the pkg.json manually so never noticed